src/components/Error/Error.module.scss

```diff
@@ -0,0 +1,58 @@
+.errorScreen {
+  position: absolute;
+  width: 100vw;
+  height: 100%;
+  background: #000;
```

Comment:
The idea, is if images can't be shared this popup appears. So you don't need to add black background.

Reply:
It makes, I wasn't sure about the background. I'm gonna make all the changes.